prisma/seed.ts

```diff
@@ -0,0 +1,38 @@
+import { PrismaClient, ProgramType } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+async function main() {
+  await prisma.student.deleteMany();
+  await prisma.program.deleteMany();
+
+  // Create a new program
+  await prisma.program.create({
+    data: {
+      id: 1,
+      title: 'Program 1',
+      description: 'Program 1 description',
+      type: ProgramType.SCIENCE,
+    },
+  });
+
+  // Create a new student
+  await prisma.student.create({
+    data: {
+      id: 1,
+      name: 'Student 1',
+      email: 'student1@gmail.com',
+      program_id: 1,
+      phone: '1234567890',
+      address: '123 Main St',
```

Comment:
Please, avoid using static values. I have placed a merge request from (feature/admins-controller-unit-test-updated) branch. Wait till @shafayat-bs23 bhai gives his review and approval. Then follow my unit test code to make it dynamic.